docs/4.2/faq.md

```diff
@@ -49,6 +49,28 @@ Teleport nodes connected to proxy server (IoT) | 2,000* | 2x 2-4 vCPUs/8GB RAM |
 
 * Teleport 4.1 release will focus on increasing Teleport IoT supported count to 10,000
 
+### Which version of Teleport is supported? 
+
+| Release       | LTS | Release Date         | Supported Until      | Minimum tsh version  | 
+| --------------|-----| -------------------- | -------------------- | -------------------- |
+| 4.2           | Yes | December 19th, 2019  | 4.3 is released      | 3.x                  |
+| 4.1           | Yes | October 1st, 2019    | October 1st, 2020    | 3.x                  |
+| 4.0           | Yes | June 18th, 2019      | June 18th, 2020      | 3.x                  |
+| 3.2           | Yes | April 1st, 2019      | April 1st, 2020      | 2.x                  |
+| 3.1           | Yes | December 12th, 2018  | December 12th, 2019  | 2.x                  |
+
+Teleport uses a semantic versioning to release updates of software. With releases taking
```

Comment:
`uses semantic versioning`

Reply:
Also, this sentence is a weird fragment. Maybe this:

`Teleport uses semantic versioning to release updates of software, with minor releases taking around 3-4 months. The team has these informal rules about releasing software:`

Reply:
As always thanks for the review Gus.